spec/services/user_active_case_count_service_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'rails_helper'
+
+describe UserActiveCaseCountService do
+
+  let(:responding_team)   { create :responding_team }
```

Comment:
Use the new mechanism: `find_or_create :foi_responding_team`

Reply:
Done